### DISCUSSION TITLE
sky beta

### DISCUSSION THREAD

**Discussion by jajaraia0** [thatskymod/Sky-CotL-Scripts#58] [Q&A]
hii! how do i use the mod for sky beta? why is error always appear when i try to open beta in canvas? 🥹 i did download the file for beta but when i log in my account, it says server error :(( please help

**Comment by jajaraia0** [comment]
> hii! how do i use the mod for sky beta? why is error always appear when i try to open beta in canvas? 🥹 i did download the file for beta but when i log in my account, it says server error :(( please help

![Screenshot_2024-01-20-01-29-26-242_com.skyhub.su.jpg](https://github.com/thatskymod/Sky-CotL-Scripts/assets/156511238/7ae795fd-185f-4e7b-b1d3-74c23e7932a5)

**Comment by thatskymod** [comment]
## Server Error: No Authentication Info

In the context of Canvas, this typically indicates the use of an incompatible login method.

To confirm, which authentication method are you using for login?

Google login is known to be broken at present. If that is the method you are using, I recommend trying an alternative. Nintendo login tends to work best. Please try this method and let me know the outcome.

Best,
thatskymod

**Reply by jajaraia0** [reply]
yes, i was using google login. thank you so much, will try that out

**Comment by thatskymod** [comment]
This question appears to be answered, so it seems appropriate to close this topic.

If you have any more questions, feel free to reach out!

**Comment by Ruki-kun** [comment]
![Screenshot_2024_0817_110837](https://github.com/user-attachments/assets/667737d0-f14d-4f77-8df7-fce41ccd4711)

**Reply by Ruki-kun** [reply]
Someone help me why I can't play
